README.md

```diff
@@ -19,13 +18,70 @@ The board is a lightweight piece of code. The TV screen is to be used in landsca
 
 ✋ Don't even ask - it's obviously written entirely in ♥️ [Swift 2.2](https://swift.org).
 
-## Widgets
+## GCBCore
+
+The small set of sources needed to load configuration, initialize and show the Board.
+
+## CocoaPods
+
+[CocoaPods](http://cocoapods.org) is a dependency manager for Cocoa projects. 
+
+You can install it with the following command:
+
+```bash
+$ gem install cocoapods
+```
+
+To integrate GCBCore into your Xcode project using CocoaPods, add this to your `Podfile`:
+
+```ruby
+pod 'GCBCore', '~> 1.0'
+```
+
+Then, run the following command:
+
+```bash
+$ pod install
+```
+
+### Usage
+
+This is how you configure Grand Central Board:
+
+```swift
+let autoStack = AutoStack()
+let scheduler = Scheduler()
+let dataDownloader = DataDownloader()
+
+let availableBuilders: [WidgetBuilding] = [
+    ImageWidgetBuilder(dataDownloader: dataDownloader),
+]
 
-### Adding a new Widget
+let configFileName = NSBundle.localConfigurationFileName
+let configurationFetching = LocalConfigurationLoader(configFileName: configFileName,
+                                                     availableBuilders: availableBuilders)
+
+let boardController = GrandCentralBoardController(scheduler: self.scheduler, stack: self.autoStack)
+
+let configurationRefresher = ConfigurationRefresher(interval: configRefreshInterval,
+                                                    configuree: boardController,
+                                                    fetcher: configurationFetching)
+```
```

Comment:
Could we add an instruction how to display the view of GCB? Like we have in `MainViewController`?